src/bun.zig

```diff
@@ -1840,12 +1837,23 @@ pub const win32 = struct {
     pub var STDERR_FD: FileDescriptor = undefined;
     pub var STDIN_FD: FileDescriptor = undefined;
 
-    pub inline fn argv() [][*:0]u8 {
-        return std.os.argv;
+    var _argv: [][*:0]u8 = &[_][*:0]u8{};
+
+    pub fn initArgv(allocator: std.mem.Allocator) !void {
+        var iter = try std.process.ArgIterator.initWithAllocator(allocator);
+        defer iter.deinit();
+        // const cmd_line_w = std.os.windows.kernel32.GetCommandLineW();
+        // var iter = try ArgIteratorWindows.init(allocator, cmd_line_w);
+        defer iter.deinit();
```

Comment:
Do we need two `defer iter.deinit();` calls (line 1844 and 1847)?

Reply:
this was a leftover, removed in latest commit 👍